testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -1222,6 +1222,21 @@ private void assertLineage(String baseQuery, Set<String> inputTables, OutputColu
         assertLineageInternal("CREATE VIEW mock.default.create_new_materialized_view AS " + baseQuery, inputTables, outputColumnMetadata);
         assertLineageInternal("INSERT INTO mock.default.table_for_output(test_varchar, test_bigint) " + baseQuery, inputTables, outputColumnMetadata);
         assertLineageInternal(format("DELETE FROM mock.default.table_for_output WHERE EXISTS (%s) ", baseQuery), inputTables);
+
+        // Test io input tables
+        assertIoInputTable("CREATE TABLE mock.default.create_io_input AS " + baseQuery, inputTables);
+        assertIoInputTable("INSERT INTO mock.default.insert_io_input(test_varchar, test_bigint) " + baseQuery, inputTables);
+        assertIoInputTable("DELETE FROM mock.default.table_for_io_input WHERE test_bigint IN (WITH s(test_varchar, test_bigint) AS (%s) SELECT test_bigint FROM s)".formatted(baseQuery), inputTables);
```

Comment:
Can we use `table_for_output` to do the assertion here ?

Reply:
Do you mean change the table name to `table_for_output`?